liquibase-core/src/main/java/liquibase/integration/commandline/Main.java

```diff
@@ -1577,23 +1574,11 @@ protected void doMigration() throws Exception {
                     //will output to stdout:
                     currentChangeLogFile = "";
                 }
-
-                File file = new File(currentChangeLogFile);
+                final PathHandlerFactory pathHandlerFactory = Scope.getCurrentScope().getSingleton(PathHandlerFactory.class);
+                Resource file = pathHandlerFactory.getResource(currentChangeLogFile);
```

Comment:
Should we also be checking the resource accessor here?

Reply:
I don't _think_ so because I could see the scenario where we try to pull a file in s3, it doesn't exist so it falls back to another resourceaccessor, then on windows it sees a file with multiple colons in the uri and throws that unexpected ":" error we've seen. Does that make sense? 🤔

Reply:
Definitely. But aren't changelogs the one place that we want to be checking the resource accessor? I admit that I am not familiar with the code you're changing, so I don't know if its relevant or not.

Reply:
No, we should not be checking the resource accessor here. It's only reading changelog files that should be using the resource accessor. Because this is mainly an output operation, this only takes a direct file path.

Thinking through the generateChangelog/diffChangeLog impact when originally wanting to expanding resourceAccessor usage was a big part of what made me say "no, let's not change our existing logic to start taking the searchPath into account without a lot more thought because it will likely be problematic"